src/startupscriptgenerator/src/python/scriptgenerator.go

```diff
@@ -224,6 +226,15 @@ func (gen *PythonStartupScriptGenerator) buildGunicornCommandForModule(module st
 	// Default to App Service's timeout value (in seconds)
 	args := "--timeout 600 --access-logfile '-' --error-logfile '-'"
 
+	pythonEnableGunicornMultiWorkers := common.GetBooleanEnvironmentVariable(consts.PythonEnableGunicornMultiWorkersEnvVarName)
+	if pythonEnableGunicornMultiWorkers {
+		// 2N+1 number of workers is recommended by Gunicorn docs.
+		// Where N is the number of CPU threads.
+		// One worker will be reading or writing from the socket while the other worker is processing a request.
+		workers := strconv.Itoa((2 * runtime.NumCPU()) + 1)
+		args = appendArgs(args, "--workers="+workers)
```

Comment:
nit: may be it would be a good practice to check for any errors for strconv.Itoa(). Though it is highly unlikely but I would call appendArgs when workers > 0.

Reply:
Thanks, I think that's a good point, will add an error checking after this part.